### PR TITLE
Use arrow function.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ function paginate (filePath, collection, fileName, files, iteratee) {
     });
 }
 
-module.exports = function(opts) {
+module.exports = (opts) => {
```
Comment (phillipj) [inline index.js:60]: Mind elaborating what we gain from doing this?

Comment (XhmikosR) [reply index.js:60]: Just consistency. I missed it the other time.

**Comment by XhmikosR** [comment]
@phillipj can you release a new version please? Maybe make it a minor one instead of a patch one.

**Comment by phillipj** [comment]
I will when I get the chance, yes.

**Comment by XhmikosR** [comment]
Thanks, no pressure, just wanted to remind it to you.

**Comment by XhmikosR** [comment]
@phillipj friendly ping :)

**Comment by phillipj** [comment]
v4.0.1 just got published 🚀

**Comment by XhmikosR** [comment]
Not bad :) https://packagephobia.now.sh/result?p=metalsmith-yearly-pagination

**Comment by phillipj** [comment]
Ohmy that looks almost too good to be true! Good initiative using those individual lodash packages instead 👌

**Comment by XhmikosR** [comment]
The real difference isn't that huge most of the time because lodash is already included by other packages in the deps tree. But still, this does reduce the module's load time, and eventually if more packages do this, it will affect the size too.

**Comment by phillipj** [comment]
> .. and eventually if more packages do this, it will affect the size too

Couldn't agree more, would be cool to see the community in general take this to consideration a bit more.

**Pull request merged**